tests/integration/main.js

```diff
@@ -380,50 +377,7 @@ test("connection resolves with current user object", t => {
 
 // Web Push Notifications
 
-test("web push notifications succeeds in registering with Beams", t => {
-  let startHasBeenCalled = false
-  let setUserIdHasBeenCalled = false
-  let setUserIdHasBeenCalledWithUserId = null
-  let setUserIdTokenProviderFetchedToken = false
-  let mockBeamsClientSDK = {
-    start: () => {
-      startHasBeenCalled = true
-      return Promise.resolve(mockBeamsClientSDK)
-    },
-    setUserId: async (userId, tokenProvider) => {
-      setUserIdHasBeenCalled = true
-      setUserIdHasBeenCalledWithUserId = userId
-      setUserIdTokenProviderFetchedToken = await tokenProvider.fetchToken(
-        userId,
-      )
-    },
-  }
-
-  const chatManager = new ChatManager({
-    instanceLocator: INSTANCE_LOCATOR,
-    userId: "alice",
-    tokenProvider: new TokenProvider({ url: TOKEN_PROVIDER_URL }),
-    beamsInstanceInitFn: () => {
-      return Promise.resolve(mockBeamsClientSDK)
-    },
-  })
-
-  chatManager
-    .connect()
-    .then(currentUser => {
-      return currentUser.enablePushNotifications()
```

Comment:
the test `chat manager succeeds to disable all web push notifications` relies on having previously enabled push notifications in this test. I'm fairly satisfied that that case is covered in the jest tests too though, so you could probably just remove that one as well? Up to you.

Reply:
I'll remove it, it's basically a duplicate